test/integration/targets/ios_acl/tests/cli/deleted.yaml

```diff
@@ -0,0 +1,68 @@
+---
+- debug:
+    msg: "Start Deleted integration state for ios_acl ansible_connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate_config.yaml
+
+- block:
+    - name: Delete attributes of provided configured interfaces
```

Comment:
Task names need to be updated I believe.

Reply:
+1